### ISSUE TITLE
Support for the Lenovo Legion Go handheld PC

### ISSUE THREAD

**Issue by clabbin** [BartoszCichecki/LenovoLegionToolkit#1015]
### Rules

- [X] I made myself familiar with the README.
- [X] I checked closed issues for similar ones and did not find a similar feature request mentioned before.
- [X] I understand that, if insufficient information was provided, my issue will be closed without an answer.

### Version

N/A

### OS

Windows 11

### Device

Lenovo Legion Go

### Is your feature request related to a problem?

I know it’s a new device but I’m really hoping to see support for this! The extra control and customization would be amazing!

### How would you like the problem to be solved?

See the above

### What alternatives have you considered?

N/A

**Comment by BartoszCichecki** [comment]
Good idea, it is a Legion in the end. I don't have access to Legion Go nor do I plan to get one, so I can't do much but if someone wants to have a go, they are most welcome.

**Comment by CasperH2O** [comment]
Hey @BartoszCichecki , amazing tool you have here. I'm one of the developers of the Handheld Companion app, we are also looking into fan control for the Legion Go. We've been told it works similar to the Legion laptops. I'll share what we will find and discover here once we got something.

**Comment by tristan-k** [comment]
> Good idea, it is a Legion in the end. I don't have access to Legion Go nor do I plan to get one, so I can't do much but if someone wants to have a go, they are most welcome.

I currently have a Legion Go available for some a limited time. If you tell me what information you need I'm happy to assist.

**Comment by krogoth2k** [comment]
> Good idea, it is a Legion in the end. I don't have access to Legion Go nor do I plan to get one, so I can't do much but if someone wants to have a go, they are most welcome.

How can we help to add some compatibility? Add performance monitor support, fix downloads etc?
